example/postgres_standalone/main.tf

```diff
@@ -6,6 +6,9 @@ module "postgres" {
   nomad_namespace   = "default"
   nomad_host_volume = "persistence"
 
+  # consul
+  consul_tags                     = ["vagrant"]
```

Comment:
Maybe `postgres` is a more descriptive tag to use here?

Reply:
Couldn't the tag also be used to differentiate between different postgres services? So setting it to `postgres` would just double up the naming. But maybe `vagrant` is not the best example either

Reply:
This is only an example, not a module default. However, `vagrant` is not a tag I would recommend using as it is not very descriptive for a postgres service. 

Regarding conflicts, I do not think that's an issue since you can provide multiple tags etc.

Reply:
That's true, I will update the value